setup.cfg

```diff
@@ -41,7 +41,7 @@ testpaths = "astrocut" "docs"
 astropy_header = true
 doctest_plus = enabled
 text_file_format = rst
-addopts = --doctest-rst
+addopts = --doctest-modules
```

Comment:
This seems like it might have different behavior. `--doctest-rst` uses pytest-doctestplus to test the code in the rst doc files I believe. Whereas `doctest-modules` tests the package's python modules and their docstrings